### PR TITLE
Automatically deploy artifact without commit tag

### PR DESCRIPTION
**Pull request by filfat**
This is an alternative solution to automatically deploying artifacts from Travis-CI, however since this required a bit of work I tried the other one first (which requires commit tags thus defeating the whole purpose of development builds).

You have two options with this one, either you can generate a Personal access token for github and encrypt it using travis encryption method, or you can add me to write access to the repo (I still wouldn't directly push, but it could be seen as a security issue for some) and I'll generate the keys.

You can see this solution in semi-action at https://github.com/filfat-Studios-AB/3ds.js/releases (hasen't worked since a few commits back due to new dependencies).

Or you could choose to dismiss this and continue to use the previous solution that I really should have tested before doing a PR for. It's up to you 😄

### CODE REVIEW

**Comment by filfat** [comment]
I can add that this build will/should fail since I/you have yet to add the api key which requires write access.

**Comment by omkarmoghe** [comment]
sorry, im not super experienced with travis, where does the apk end up?

**Comment by filfat** [comment]
@omkarmoghe https://github.com/omkarmoghe/Pokemap/releases

**Comment by omkarmoghe** [comment]
@filfat oh fuck yeah. is it possible to tag the dev branch apks as alpha or beta or even jut "dev". maybe in the filename or something?

**Comment by filfat** [comment]
@omkarmoghe they automatically gets tagged as pre-release https://github.com/omkarmoghe/Pokemap/pull/44/files#diff-354f30a63fb0907d4ad57269548329e3R30 :)

(Provided that I've configured everything correctly, still need write access to fully test)

**Comment by omkarmoghe** [comment]
@filfat can i just build to test it out? otherwise i can add you as a collab. lmk

**Comment by omkarmoghe** [comment]
@filfat just merged a different pr - you should rebase/pull latest to resolve the conflicts real quick :)

**Comment by filfat** [comment]
@omkarmoghe Unless you know how to generate the keys (or feel like reading up) you can add me as a collaborator. I'll rebase the pr shortly.

**Comment by omkarmoghe** [comment]
@filfat ill add you. build is failing though

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ jdk:
 - oraclejdk8
 android:
   components:
-<<<<<<< fbb04dce928808017e0087ad14022fc2523f2fbf
```
Comment (omkarmoghe) [inline .travis.yml]: lolol gotta love merge conflicts

Comment (omkarmoghe) [inline .travis.yml]: lolol gotta love merge conflicts

**Comment by filfat** [comment]
@omkarmoghe Yeah, I forgot to remove a line from the yml file 😕 https://github.com/omkarmoghe/Pokemap/pull/44/commits/8b84d8c2bacd18d57e9a24a5fd21b3928dcbafc3#diff-354f30a63fb0907d4ad57269548329e3L6

**Comment by omkarmoghe** [comment]
@filfat added as collabo

**Comment by filfat** [comment]
Alright, I'll see if anything fails (should work provided I configured the keys correctly otherwise I'll just have to do it  again).

**Comment by omkarmoghe** [comment]
dope

**Comment by filfat** [comment]
@omkarmoghe I will need to have write access though due to the script temporary pushing to a branch https://github.com/omkarmoghe/Pokemap/blob/dev/.travis.yml#L35

**Comment by omkarmoghe** [comment]
accept the invite haha

**Comment by filfat** [comment]
Had to refresh to see that *facepalm* 😅

**Comment by filfat** [comment]
As I thought, the keys were wrong; give me a second to update them.

**Comment by filfat** [comment]
@omkarmoghe https://github.com/omkarmoghe/Pokemap/releases works perfectly now!
I'm going to go to bed now and try to get the 5 remaining hours of sleep 🎉

**Comment by omkarmoghe** [comment]
yeeeeee thanks boss

**Pull request merged**